layouts/partials/components/giscus.html

```diff
@@ -1,4 +1,26 @@
-{{- $lang := site.Language.LanguageCode | default `en` -}}
+{{ $default_chinese := "zh-CN" }}
+
+{{ $sl := site.Language }}
+{{ $giscus_lang := $sl.Lang | default `en` }}
+
+{{/* 
+  Special case for Chinese.
+  Giscus uses the geophraphical language code for these.
+  See: https://github.com/giscus/giscus/tree/main/locales
+  */}}
+{{ if eq $giscus_lang "zh" }}
```

Comment:
In this case, `$giscus_lang` could also be `zh-cn`, so it's better to use `hasPrefix` instead of `eq`.